subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidStudioMockupPerformanceTest.groovy

```diff
@@ -33,8 +35,12 @@ class RealLifeAndroidStudioMockupPerformanceTest extends AbstractCrossVersionPer
 
     def "get IDE model for Android Studio"() {
         given:
+        runner.args = [AndroidGradlePluginVersions.OVERRIDE_VERSION_CHECK]
         def testProject = AndroidTestProject.projectFor(runner.testProject)
         testProject.configure(runner)
+        if (testProject instanceof IncrementalAndroidTestProject) {
+            IncrementalAndroidTestProject.configureForLatestAgpVersionOfMinor(runner, AbstractRealLifeAndroidBuildPerformanceTest.SANTA_AGP_TARGET_VERSION)
+        }
```

Comment:
Can we just add a no-op implementation of this method in non-`IncrementalAndroidTestProject` instead of doing an `instanceof`?

Reply:
I did something different: I moved the static methods to `AndroidTestProject`.